README.md

```diff
@@ -91,10 +91,12 @@ The easiest way to run releasetool is via docker.
 2. Copy `zaptain_ui/settings_template.py` to `zaptain_ui/settings_local.py` and adjust the File to your needs.
 Especially change the `SECRET_KEY` variable and remove the line below it.
 One possibility to create a secret is `< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c${1:-64};echo;`.
+The variable `MEDIA_ROOT` contains the location of the database file. and `$MEDIA_ROOT/releasetool` will contain the releasecandidates.
 3. Run `docker build -t rt .` from inside the base directory of this repository.
 4. Run `docker run -it --rm -p 8085:8000 rt`.
 To persists the data use [docker volumes](https://docs.docker.com/storage/volumes/).
-The important directorys inside the container are `/rt_data` and `rt_media`.
```

Comment:
``/rt_data`` should be bound to a local folder in order to persist the database.
``/releasetool/$MEDIA_ROOT/`` should be bound to a local folder in order to persist the release candidates.